pkg/addonfactory/addonfactory.go

```diff
@@ -76,6 +79,12 @@ func (f *AgentAddonFactory) WithAgentRegistrationOption(option *agent.Registrati
 	return f
 }
 
+// WithTrimCRDs is to enable trim the CRDs in manifestWork.
+func (f *AgentAddonFactory) WithTrimCRDs() *AgentAddonFactory {
```

Comment:
`WithTrimCRDDescription`? to clarify that we're removing merely descriptions.

Reply:
changed to WithTrimCRDDescription